src/data/html-quiz.ts

```diff
@@ -1,4 +1,34 @@
 const htmlQuiz = [
+  {
+    Question: "How many Heading tag are there in HTML5",
+    Answer: "6",
+    Distractor1: "4",
+    Distractor2: "8",
+    Distractor3: "6",
```

Comment:
distractors are supposed to be for the incorrect answers

```suggestion
    Distractor3: "1",
```